### ISSUE TITLE
Aliucord not initializing and emotes not loading

### ISSUE THREAD

**Issue by nin0-dev** [Aliucord/Aliucord#376]
### Discord Account

ohiostan

### What happens when the bug or crash occurs?

this bug is preventing Aliucord to load, as in the app loads, I can use it, but no plugins or themes, and for some reason, emotes (builtin) do not load, while they do on vanilla...
[as note, i am on latest android 14 beta, so it may be related to that if you cant reproduce the bug on a13 stable]

### What is the expected behaviour?

I expect aliucord to initialize and be able to use it, and view and send emojis

### How do you recreate this bug or crash?

1. install aliucord with latest version of installer/app
2. see by yourself

### Crash-log

no crashlog but the error logcat is
```
[[[[[at startup]]]]]
07-31 08:50:21.017 E/com.aliucord(24023): Not starting debugger since process cannot load the jdwp agent.
07-31 08:50:21.086 E/Injector(24023): Failed to initialize Aliucord
07-31 08:50:21.086 E/Injector(24023): java.lang.UnsatisfiedLinkError: JNI_ERR returned from JNI_OnLoad in "/data/app/~~EkA-HC6B3PVsIp20mlsY4w==/com.aliucord-S5n1uyOEQYbHIYFa33XQ-Q==/lib/arm64/libaliuhook.so"
07-31 08:50:21.086 E/Injector(24023): 	at com.aliucord.injector.InjectorKt.init(Injector.kt:41)
[[[[[when trying to load an emoji]]]]]
07-31 08:50:38.457 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.558 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.559 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.560 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.564 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.570 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.573 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.580 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.585 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:41.586 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:42.933 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:42.963 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:42.964 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:42.965 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:43.032 E/com.aliucord(24023): Invalid resource ID 0x00000000.
07-31 08:50:43.130 E/com.aliucord(24023): Invalid resource ID 0x00000000.

```

### Request Agreement

- [x] I did indeed check to make sure the bug or crash report is applicable!

**Comment by nin0-dev** [comment]
(clarifying the emotes do work on stock 126.21 discord, even on a14, which leads me to think its an Aliucord issue atp)

**Comment by Juby210** [comment]
Fixed in 03a64a4b0faf77d169871fc9ca13ec220e0b1056
